### DISCUSSION TITLE
Multi objective bayesian optimization(max-min pareto front)

### DISCUSSION THREAD

**Discussion by YouHyin** [meta-pytorch/botorch#1498] [Q&A]
I need to maximize one of the two functions, and minimize the other.   

    for _ in range(n_candidates):
        weights = sample_simplex(2).squeeze()
        objective = GenericMCObjective(
            get_chebyshev_scalarization(
                weights,
                pred
            )
        )
        acq_fun = qNoisyExpectedImprovement(
            model=model,
            objective=objective,
            sampler=sampler,
            X_baseline=train_x,
            prune_baseline=True,
        )
        acq_fun_list.append(acq_fun)

    candidates, _ = optimize_acqf_list(
        acq_function_list=acq_fun_list,
        bounds=standard_bounds,
        num_restarts=NUM_RESTARTS,
        raw_samples=RAW_SAMPLES,
        options={
            "batch_limit": 5,
            "maxiter": 200,
        }
    )

    return unnormalize(candidates, bounds)

I want to perform max-min Bayesian optimization using GenericMCOjective, how can I modify this code? Or is there any other way?

I would appreciate it if you could tell me how to draw the max-min paleto front.

![max-min](https://user-images.githubusercontent.com/113442395/201479506-8e925678-8853-4ad5-a13b-31f93b61c72d.png)

**Comment by Balandat** [comment]
The most straightforward thing would is to just model the outcomes with the sign of the to-be-minimized objective to be flipped. You can then compute the PF (assuming maximization) and flip the sign of the outcome again afterwards. 

A separate comment about your approach: You are generating a list of acquisition functions and then generate candidates from them, but you are not taking into account the effect of the "pending" points you have selected in the earlier optimizations when selecting the later ones. The proper thing to do here is not use `optimize_acqf_list` but use `optimize_acqf` sequentially, and generate the acquisition functions with the different weights while passing the previously generated (not evaluated) candidates in via `X_pending`.
